doc/design/test/nsx.md

```diff
@@ -0,0 +1,49 @@
+# NSX Initial testing notes
+
+##Required HW setup:
+- vSphere 6.0 Cluster setup
+- An ESX host in the cluster with a minimum of 4 CPU to host the NSX Manager Appliance
+
+##Initial install:
+
+##Deploy a Nimbus Cluster using 5-2-Cluster test.
```

Comment:
triple hash the following lines, otherwise it looks weird being on the same level as Initial install.